app/package.json

```diff
@@ -138,6 +136,7 @@
 		"micromustache": "8.0.3",
 		"mime": "3.0.0",
 		"mitt": "3.0.0",
+		"modern-diacritics": "2.3.1",
```

Comment:
I have my doubts about this packages. No downloads, no indication of maintenance. I know the original diacritics hasn't been updated in what feels like centuries, but it still has a huge amount of people checking/relying on it 🤔

Reply:
>  I know the original diacritics hasn't been updated in what feels like centuries, but it still has a huge amount of people checking/relying on it 🤔

That's sums up the problem of the Node.js ecosystem quite well, if you ask me 🤣 People just don't care about outdated packages, so I'm not sure whether this number should be that decisive. I took a glance at `modern-diacritics` and it made a solid impression to me. If for example a security bug should appear, I also think that the chances are higher that the newer package will receive a fix than the package that has not received any updates for 7 years.
But I get your doubts, I'm always checking number of downloads too!

![Screenshot 2023-08-18 at 20-16-17 diacritics vs fold-to-ascii vs modern-diacritics vs normalize-diacritics npm trends](https://github.com/directus/directus/assets/5363448/dc9298e8-5e58-4b6a-9abf-386eae285663)
https://npmtrends.com/diacritics-vs-fold-to-ascii-vs-modern-diacritics-vs-normalize-diacritics

We could also take a look at [`normalize-diacritics`](https://www.npmjs.com/package/normalize-diacritics
).

Let me know what you prefer!

Reply:
A third option is: this library really doesn't do anything from a find and replace on a list of hardcoded diacritics characters. Almost wondering if it makes more sense to just roll that ourselves and remove the dependency on third party software altogether

Reply:
Right, also played with this thought! 👍

Reply:
Will be addressed in a separate PR 👍